project/src/DataBase.q

```diff
@@ -1,48 +1,65 @@
 / Initialize data table.
-t:([]sym:enlist`MSFT;bid:enlist 50;bidSize:enlist 10;ask:enlist 80;askSize:enlist 30;time:.z.p)
+t:([]sym:enlist`MSFT;bid:enlist 50;bidSize:enlist 10;ask:enlist 80;askSize:enlist 30;time:.z.p) //! Temporary
 
-//! Needs documentation.
-bidGen:{[lastBid;lastAsk]
-	$[0=lastAsk-lastBid;
-		first[t`bid]+floor 0.5+rand[0.1*a]-rand 0.1*a:first t`bid;
-		lastBid+rand ceiling[rand abs 0.1*1+lastAsk-lastBid],0]
+/ Gets a file handle.
+/ @param relPath	{string}	Relative path (from project root) to file.
+getFileHandle:{[relPath]
+	hsym`$getenv[`DB_PROJECT_HOME],"/",relPath
  }
 
-//! Needs documentation.
-bidSizeGen:{[lastBid;lastAsk]
-	$[0=lastAsk-lastBid;
-		last[t`bidSize]+floor 0.5+rand[0.1*last t`bidSize]-rand 0.1*last t`bidSize; 
-		last t`bidSize]
+/ Reads variance parameters from params.csv.
+cfg:1!("SFCS";enlist",")0:getFileHandle"cfg/params.csv"
```

Comment:
This should be in its own file and have the values stored in the `.cfg` namespace. Every file you write is going to need this, so better to have it in a central place.